### PR TITLE
Remove outdated build tools

### LINKED ISSUE #55: Remove haw
We last updated `haw` in 2017.
https://github.com/zooniverse/haw

Let’s remove it and maintain this repo as a HTML and CSS file. If a static side builder would make updates easier, Eleventy would be a good replacement for `haw`.

### PR DESCRIPTION
**Pull request by eatyourgreens**
- Remove `haw` and `publisssh`.
- Install `serve` for local testing.
- Remove the build step from deployment.

Closes #55.

Should close all the open security alerts too, which are all vulnerabilities in the build tools.

### CODE REVIEW

File: .github/workflows/deploy_production.yml [hunk 1]
```diff
@@ -10,18 +10,9 @@ on:
...
-    uses: zooniverse/ci-cd/.github/workflows/npm_build.yaml@main
-    with:
-      commit_id: ${{ github.sha }}
-      node_version: '16.x'
-      output: 'build'
-      script: 'build'
   deploy_production:
     name: Deploy production
     uses: zooniverse/ci-cd/.github/workflows/deploy_static.yaml@main
-    needs: build
```
Comment (eatyourgreens) [inline .github/workflows/deploy_production.yml:24]: On second thoughts, I think the deploy downloads the HTML and CSS files from the build step, so this should stay and we should have a build step that uploads the HTML and CSS.

**Comment by shaunanoordin** [comment]
Just to confirm that I'm understanding this PR correctly:

- We're removing the `build` step from the deploys (also the `npm run build` command is no longer needed)
- This means there's no "source code" vs "compiled code" any more, just the raw non-compiled code that's ready to deploy
- This non-compiled code sits in the `/build` folder
- The only practical effect is that `main.styl` is "permanently transmogrified" into `main.css`. So, as long as we don't need to edit the styles too much, this change will have almost no significant impact.

OK, I'm down with this.

Thoughts:
- If we're going with non-compiled code, should that `build/main.fc8ca9.css` file just be called `build/main.css` ?
- [very minor nitpick][safe to ignore] Similarly, shouldn't that folder now be called `/app` or `/src` instead of `/build` since we're not building anything? (Wait, _a build_ is also a noun describing an incarnation of code, so _build_ doesn't necessarily imply the verb. Hmmm)
- I think we can add `on: pull_request:` (or something like that) to the staging deploy to test if/confirm that the deployment pipeline works without that `build` step

I think the only thing I'd really like to see is the deploy_staging code getting a test run in this PR, so we can confirm that the deploy looks good on https://jobs.preview.zooniverse.org/

**Comment by eatyourgreens** [comment]
>The only practical effect is that main.styl is "permanently transmogrified" into main.css. So, as long as we don't need to edit the styles too much, this change will have almost no significant impact.

To be honest, the Jobs page has minimal styling. There's very little difference between the stylus file and the final CSS.

>If we're going with non-compiled code, should that build/main.fc8ca9.css file just be called build/main.css?

Depends what cche headers we're sending with that CSS file. If we want cache-busting, then it should be renamed each time it's edited. Which implies some sort of minimal build step.  We last edited those styles 7 years ago so 🤷‍♂️.

>[very minor nitpick][safe to ignore] Similarly, shouldn't that folder now be called /app or /src instead of /build since we're not building anything? (Wait, a build is also a noun describing an incarnation of code, so build doesn't necessarily imply the verb. Hmmm)

If I rename it to `/docs`, then we can preview its contents using GH Pages. That might be useful.

>I think we can add on: pull_request: (or something like that) to the staging deploy to test if/confirm that the deployment pipeline works without that build step.

The deployment workflow downloads the files that it deploys, so we are going to need some sort of minimal build step, even if it's just a step that uploads the HTML and CSS file.
https://github.com/zooniverse/ci-cd/blob/ca8a9f8c975d0af067af07e2d6aa0eb0a4baa0bc/.github/workflows/deploy_static.yaml#L30-L33

**Comment by eatyourgreens** [comment]
@shaunanoordin I've run a test deploy from this branch:
https://github.com/zooniverse/jobs.zooniverse.org/actions/runs/2147722467

**Pull request merged**